src/ess/powder/conversion.py

```diff
@@ -229,8 +237,35 @@ def convert_reduced_to_tof(
     )
 
 
-def convert_monitor_do_wavelength(
-    monitor: MonitorData[RunType, MonitorType],
+def compute_detector_time_of_flight(
+    detector_data: DetectorData[RunType], tof_workflow: TofWorkflow
+) -> TofData[RunType]:
+    wf = tof_workflow.pipeline.copy()
```

Comment:
How expensive is this? Does it copy the cached intermediate results?

Reply:
I always assumed it made a cheap shallow copy?
I think it's using https://networkx.org/documentation/stable/reference/classes/generated/networkx.Graph.copy.html?